crates/re_viewer_context/src/space_view/view_query.rs

```diff
@@ -20,25 +26,12 @@ pub struct ViewQuery<'s> {
     /// The entity properties for all queried entities.
     /// TODO(jleibs/wumpf): This will be replaced by blueprint queries.
     pub entity_props_map: &'s EntityPropertyMap,
+
+    /// Hover/select highlighting information for this space view.
+    pub highlights: &'s SpaceViewHighlights,
```

Comment:
Moving highlights into the `ViewQuery` feels a bit odd to me. What is highlighted is *context* -- it shouldn't govern how the query operates. Should this be pushed up into `ViewerContext` instead?

Reply:
agreed, but the highlights are precomputed per space view (this makes a few things a bit simpler and faster). So the alternative is to have this as a separate field on PartSystem and SpaceViewClass::ui 🤔

Reply:
huh. we could make it a `ViewContextSystem` 🤔 though!
Not doing that now but leaving a todo in the code